src/controllers/SubscribeController.js

```diff
@@ -3,7 +3,7 @@ import SubscribeService from '../services/SubscribeService';
 
 const subscribe = async (req, res) => {
   GenericResponse.sendAcknowledgement(res);
-  SubscribeService.subscribe(req.body.data.message);
+  SubscribeService.subscribe(req.body.message);
```

Comment:
swap the lines. You first do the work then ack

Reply:
Updated